lib/src/components/NxSearchTransferList/__tests__/NxSearchTransferList.test.tsx

```diff
@@ -28,6 +28,37 @@ describe('NxSearchTransferList', function() {
       quickRender = rtlRender(NxSearchTransferList as ComponentType<Props<string>>, minimalProps),
       renderEl = rtlRenderElement(NxSearchTransferList as ComponentType<Props<string>>, minimalProps);
 
+  beforeEach(function() {
```

Comment:
Oh right, I forgot this was copied into several different test files.  Please move it to a utility file in `lib/src/__testutils__`

Reply:
> Oh right, I forgot this was copied into several different test files. Please move it to a utility file in `lib/src/__testutils__`

done 94307d7e404072e402735204b7137b776a18773f